source/eks.rst

```diff
@@ -0,0 +1,165 @@
+==========================================================================================
+Install Percona XtraDB Cluster on Amazon Elastic Kubernetes Service (EKS)
+==========================================================================================
+
+This quickstart shows you how to deploy Percona XtraDB cluster operator on Amazon Elastic Kubernetes Service (EKS). The document assumes some experience with Amazon EKS. For more information on the EKS, see the `Amazon EKS official documentation <https://aws.amazon.com/eks/>`_.
+
+Prerequisites
+=============
+
+The following tools are used in this guide and therefore should be preinstalled:
+
+1. **AWS Command Line Interface (AWS CLI)** for interacting with the different
+   parts of AWS. You can install it following the `official installation instructions for your system <https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html>`_.
+
+2. **eksctl** to simplify cluster creation on EKS. It can be installed
+   along its `installation notes on GitHub <https://github.com/weaveworks/eksctl#installation>`.
+
+3. **kubectl**  to manage and deploy applications on Kubernetes. Install
+   it `following the official installation instructions <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_.
+
+Create a new IAM role with EKS permissions
+===========================================
+
+The first thing to use is configuring the `AWS-IAM-Authenticator <https://github.com/kubernetes-sigs/aws-iam-authenticator>`_ to allow IAM authentication with your Kubernetes cluster.
+
+Open the IAM console, select Roles on the left and then click the Create Role button at the top of the page.
+From the list of AWS services, select EKS and then Next: Permissions at the bottom of the page.
+Leave the selected policies as-is, and proceed to the Review page. Enter a name for the role (e.g. eksrole) and hit the Create role button at the bottom of the page to create the IAM role. The Role ARN specified on the page is to be used later.
```

Comment:
is it really needed?
I think it is just needed to configure aws credentials according to https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html doc is enough